modules/administration/pages/maintenance-window.adoc

```diff
@@ -1,33 +1,39 @@
 [[maintenance-window]]
-= Maintenance Window Checklist
+= {productname} Maintenance Window
 
-If you work on a {productname} Server with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+If you work with scheduled maintenance windows, you might find it difficult to remember all the things that you need to do before, during, and after that critical downtime of the {productname} Server.
+{productname} Server related systems such as Inter-Server Synchronization Slave Servers or {productname} Proxies are also affected and have do be considered.
+
+{suse} recommends to keep the {productname} infrastructure (servers, proxies, build hosts, etc.) always up-to-date.
+Because the {productname} Server is needed to update all your IT landscape, the server must always be updated.
+// It's similar to zypper at the package level:
+You must keep it up-to-date or you will risk not being able to update something else successfully!
 
-As with any {sle} system, you are encouraged to install package updates on the {susemgr} Server when they arrive.
 
-This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
 
+== {productname} Server
+
+This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
 
-. Stop services.
-You will need to stop the spacewalk, SAP, and database services, along with any others you have running.
+// ke, 2019-09-30: we'll stop spacewalk during the update
+// . Stop spacewalk services.
+// You will need to stop the spacewalk, SAP, and database services, along with any others you have running.
 . Check if the configuration is still correct.
-. On Salt systems, apply the highstate.
 . Apply the latest updates.
 See xref:upgrade:server-update.adoc[].
 . Update to the latest service pack, if required.
 See xref:upgrade:migrate-4x-4x.adoc[].
 . Reboot the server.
 . Check if the configuration is still correct.
-. On Salt systems, apply the highstate.
-. Start any stopped services.
+// . Start any stopped services.
```

Comment:
I am confused why this is commented now.

Reply:
FMPOV this should be removed, and made part of the update/migration procedures.

Here we could mention, maybe, that the user should use `spacewalk-service status` and check if all the services required are up and running (problem: not all users require the same services).

Reply:
i'm now mentioning this.  let's see whether people will like it.